### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/data2/test/slt/tnb/script_test/target243/MHC_Class_I/tmp/TUMOR.ann.HLA-A*30:01.9.tsv_1-4'

### ISSUE THREAD

**Issue by snowsl** [griffithlab/pVACtools#446]
* pvactools version:1.5.0
* Python version:3.6.5
* Operating System:Linux

**Describe the bug**
hi,
i split my big vcf file into hundreds of small one and execute them separately, others result is fine, only this one get error.  i think that empty fasta files caused this error. There may be some circumstances not considered.
please help to check that 
**To Reproduce**
```
pvacseq run ./target243.fvep.readcount.gx.tx.dp.vcf TUMOR 'HLA-A*30:01,HLA-A*32:01,HLA-B*52:01,HLA-B*52:07,HLA-C*06:02,HLA-C*12:02' NetMHC NetMHCpan PickPocket NetMHCcons ./ -e 9 --normal-sample-name NORMAL --downstream-sequence-length 100 --iedb-install-directory /opt/iedb/ --fasta-size 500
```
[target243.fvep.readcount.gx.tx.dp.vcf.zip](https://github.com/griffithlab/pVACtools/files/3495734/target243.fvep.readcount.gx.tx.dp.vcf.zip)

**Log Output**
```Executing MHC Class I predictions
Converting .vcf to TSV
Completed
Splitting TSV into smaller chunks
Splitting TSV into smaller chunks - Entries 1-2
Completed
Generating Variant Peptide FASTA and Key Files
Generating Variant Peptide FASTA and Key Files - Entries 1-4
Completed
Fasta file /data2/test/slt/tnb/script_test/target243/MHC_Class_I/tmp/TUMOR_21.fa.split_1-4 is empty. Skipping
Parsing binding predictions for Allele HLA-A*30:01 and Epitope Length 9 - Entries 1-4
Parsing prediction file for Allele HLA-A*30:01 and Epitope Length 9 - Entries 1-4
Traceback (most recent call last):
  File "/opt/conda/bin/pvacseq", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.6/site-packages/tools/pvacseq/main.py", line 92, in main
    args[0].func.main(args[1])
  File "/opt/conda/lib/python3.6/site-packages/tools/pvacseq/run.py", line 164, in main
    pipeline.execute()
  File "/opt/conda/lib/python3.6/site-packages/lib/pipeline.py", line 440, in execute
    split_parsed_output_files = self.parse_outputs(chunks)
  File "/opt/conda/lib/python3.6/site-packages/lib/pipeline.py", line 398, in parse_outputs
    parser.execute()
  File "/opt/conda/lib/python3.6/site-packages/lib/output_parser.py", line 384, in execute
    iedb_results = self.process_input_iedb_file(tsv_entries)
  File "/opt/conda/lib/python3.6/site-packages/lib/output_parser.py", line 317, in process_input_iedb_file
    iedb_results = self.parse_iedb_file(tsv_entries)
  File "/opt/conda/lib/python3.6/site-packages/lib/output_parser.py", line 489, in parse_iedb_file
    with open(input_iedb_file, 'r') as reader:
FileNotFoundError: [Errno 2] No such file or directory: '/data2/test/slt/tnb/script_test/target243/MHC_Class_I/tmp/TUMOR.ann.HLA-A*30:01.9.tsv_1-4'

```

**Output File**
If your bug is related to the contents of an output file, please paste the output file here.

**Expected behavior**

**Comment by susannasiebert** [comment]
Thank you for reporting this bug. It has been fixed in version 1.5.1. Please give the new version a tray and let me know you run into any additional errors.

**Comment by snowsl** [comment]
everything is fine by the new version，thanks
